src/main/java/com/ericsson/ei/frontend/utils/BackendInformationControllerUtils.java

```diff
@@ -0,0 +1,95 @@
+/*
+   Copyright 2018 Ericsson AB.
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.ericsson.ei.frontend.utils;
+
+import java.net.URI;
+import java.util.stream.Collectors;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+import com.google.gson.JsonArray;
+
+@Component
+public class BackendInformationControllerUtils {
```

Comment:
You may remove the Utils part of the name since the package is utils

Reply:
But it's not a controller either. This part as i mentioned before should be refactored with both name and functionality later.

Reply:
I guess we can keep the Utils in the className. since it is a utility package we can have n number of utils classes.